src/pages/_proxied-dot-io/nomad/docs/content.json

```diff
@@ -0,0 +1,122 @@
+{
+  "pageTitle": "Documentation",
+  "pageSubtitle": "Welcome to the Nomad documentation. Nomad is a scheduler and workload orchestrator. This documentation is a reference for all available features and options of Nomad.",
+  "featuredCard": {
+    "heading": "Get Started",
+    "body": "Learn how to use Nomad to schedule and orchestrate workloads.",
+    "links": [
+      {
+        "title": "CLI Quick Start",
+        "url": "https://learn.hashicorp.com/collections/nomad/get-started"
+      }
+    ],
+    "image": {
+      "src": "https://www.datocms-assets.com/2885/1649993001-nomad-featured-card-image.png",
+      "alt": ""
+    }
+  },
+  "useCaseCards": [
+    {
+      "heading": "Nomad Fundamentals",
+      "body": "Become familiar with the core concepts of Nomad.",
+      "links": [
+        {
+          "title": "Installing Nomad",
+          "url": "/docs/install"
+        },
+        {
+          "title": "Writing Job Specs",
+          "url": "/docs/job-specification"
+        },
+        {
+          "title": "Agent Configuration",
+          "url": "/docs/configuration"
+        },
+        {
+          "title": "Deploy Nomad Jobs",
+          "url": "https://learn.hashicorp.com/tutorials/nomad/jobs?in=nomad/manage-jobs"
+        }
+      ]
+    },
+    {
+      "heading": "Nomad Pack",
+      "body": "Use Nomad Pack to deploy popular applications to Nomad.",
+      "links": [
+        {
+          "title": "Introduction to Nomad Pack",
+          "url": "https://learn.hashicorp.com/tutorials/nomad/nomad-pack-intro?in=nomad/nomad-pack"
+        },
+        {
+          "title": "Write a custom Nomad Pack",
+          "url": "https://learn.hashicorp.com/tutorials/nomad/nomad-pack-writing-packs?in=nomad/nomad-pack"
+        },
+        {
+          "title": "Nomad Pack repository",
+          "url": "https://github.com/hashicorp/nomad-pack-community-registry"
+        }
+      ]
```

Comment:
Maybe also?

Community Registry
https://github.com/hashicorp/nomad-pack-community-registry

(or "Community Pack Registry" if it fits?)

Reply:
Updated